src/controllers/commentController.js

```diff
@@ -1,7 +1,8 @@
 import db from '../database/models/index';
 import sendResult from '../utils/sendResult';
+import CommentService from '../services/comment.service';
 
-const comment = {
+const CommentController = {
```

Comment:
Let's maintain pluralized names for the controllers. So this should be `CommentsController`